constants.xml

```diff
@@ -6,6 +6,7 @@
         <DS_MULTICAST_CLUSTER_SIZE>10</DS_MULTICAST_CLUSTER_SIZE>
         <TX_SHARING_CLUSTER_SIZE>10</TX_SHARING_CLUSTER_SIZE>
         <COMM_SIZE>200</COMM_SIZE>
+        <!-- Only take effect when shard node number < 651-->
```

Comment:
Can you put this comment in the same line as COMM_SIZE? Although I'm not sure if clang-tidy will reformat it.

Reply:
Actually clang-tidy/clang-format will check this, but anyway I will move the comment above <COMM_SIZE>.